orders/src/main/java/com/orders/crudyorders/model/Agent.java

```diff
@@ -0,0 +1,104 @@
+package com.orders.crudyorders.model;
+
+import javax.persistence.*;
+import java.util.ArrayList;
+
+@Entity
+@Table(name = "agent")
+public class Agent
```

Comment:
Line 7 - It is advisable to make table names plural in order to reduce the likelihood of naming conflicts (the class name below can still be singular).